README.adoc

```diff
@@ -938,6 +938,19 @@ User.where("id != ?", id)
 User.where.not(id: id)
 ----
 
+=== Finding orphan records [[finding-orphan-records]]
+
+If you're using Rails 6.1 or higher, use https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods/WhereChain.html#method-i-missing[where.missing] to find orphan records.
```

Comment:
I would change the phrasing slightly.
Those are not always orphan records, implying that there's a deficiency in the FKs in the DB schema.

There could be legitimate cases for a record not to have an associated record, e.g. order->delivery (no delivery scheduled because the order is not paid yet), node->parent (root node), post->comments (no comments yet).

Reply:
Indeed it is. How about "Finding records with nothing associated records" for a different expression?

Reply:
How about mirroring the terminology from the [docs](https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods/WhereChain.html#method-i-missing), e.g.

> to identify missing relations.

(you could argue that 'missing' also suggests a deficiency, but I think it's better than 'orphaned').

Reply:
I think it looks good.
I changed it, how do you like it?